### DISCUSSION TITLE
Most Stable Target MCU / Dev Board?

### DISCUSSION THREAD

**Discussion by thestumbler** [modm-io/modm#1159] [General]
I've inherited a prototype project implemented on an Arduino, and refactored the code into an object-oriented C++ style (per request of client). Next up, the ATMega328P is on the edge of being not powerful enough for the task. Ultimately this MCU will be replaced with a much more modern chip in a completely new PCB design. In the meantime, I would like to use an off-the-shelf development board for some tests.  After my previous struggles with MODM on an STM32H7 project, I'd like to pick a target that is very well supported in MODM. 

I'm thinking of an RP2040 or one of the STM32F4-based dev boards. What would you guys nominate as the best supported MODM dev board of all time?

**Comment by salkinium** [comment]
Well, anything that's not a STM32F7/H7 is quite well supported ;-)

The STM32 NUCLEO boards don't contain a lot of on-board hardware, so they are a safe bet.
The STM32G0 series is quite cheap and probably powerful enough to replace a ATmega (and more modern than F0/L0), otherwise go with a F4 device.

(RP2040 support in modm is quite basic and not as good as the official library.)
